deploy/eck-stack/templates/tests/kibana_test.yaml

```diff
@@ -0,0 +1,48 @@
+suite: test kibana
+templates:
+  - charts/eck-kibana/templates/kibana.yaml
```

Comment:
Same here regarding dependency build as with the Elasticsearch test and I would move them to the Kibana chart and have only tests that test the stack chart here.